src/views/AddPriceSingle.vue

```diff
@@ -136,10 +135,19 @@
 </template>
 
 <script>
+import Compressor from 'compressorjs'
 import api from '../services/api'
 import BarcodeScanner from '../components/BarcodeScanner.vue'
 import LocationSelector from '../components/LocationSelector.vue'
 
+Compressor.setDefaults({
+  checkOrientation: true,  // default
+  retainExif: true,
+  quality: 0.6,
+  // mimeType: 'image/webp',
```

Comment:
@raphael0202 I commented it out because I have issues when the config is set to webp: the image is compressed, and the output is displayed, but it stored without an extension (None instead). It's probably due to the backend image mgmt, so we'll need a seperate PR to fix it ;)

Reply:
done here ! https://github.com/openfoodfacts/open-prices-frontend/pull/44